atomic_defi_design/qml/Components/DexBip39WordListChecker.qml

```diff
@@ -0,0 +1,2061 @@
+import QtQuick 2.15
```

Comment:
there is mnemonic_validate function for that https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/atomic_defi_design/qml/Screens/RecoverSeed.qml#L157